packages/terra-core-docs/src/terra-dev-site/doc/button/About.1.doc.mdx

```diff
@@ -23,6 +23,11 @@ Submit a request if additional variants not provided are needed.
 
 The `action` variant is intended for specific solutions in which a non-inline floating button sits above the page content in a fixed position.
 
+### Assistive Technology Support
+#### WAI ARIA Roles and States
+* [aria-pressed](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed)
+    * With Safari and Chrome, VoiceOver screenreader is partially compatible with aria-pressed. When aria-pressed is set to true, VoiceOver announces the selection state; when aria-pressed is set to false, it fails to announce the de-selection state.
```

Comment:
Let's move this section below component features for consistency

Reply:
https://github.com/cerner/terra-core/pull/3999/commits/5cac5a11f105f0280bd534aa5160a2211c4aac50